content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -68,6 +68,17 @@ $ export CTX_REMOTE_CLUSTER=remote_cluster
 $ export REMOTE_CLUSTER_NAME=remote_cluster
 {{< /text >}}
 
+{{< warning >}}
+If you are using [kind](/docs/setup/platform-setup/kind/) cluster for testing external control plane, make sure to use the following names.
+
+{{< text bash >}}
+$ export CTX_EXTERNAL_CLUSTER=kind-external-cluster
```

Comment:
can we just change this to external-cluster (without kind)?  didn't want the instruction to focus on kind :)

Otherwise LGTM.

Reply:
I also don't want to append it but I think that's how `kind` works. it automatically appends `kind-` to the cluster name.

```
kind create cluster --name external-cluster
```
```
$ kubectl config view
...
contexts:
- context:
    cluster: kind-external-cluster
    user: kind-external-cluster
  name: kind-external-cluster
- context:
    cluster: kind-remote-cluster
    user: kind-remote-cluster
  name: kind-remote-cluster
current-context: kind-remote-cluster
```

Reply:
The fact that kind appends `kind-` is well documented in the kind documentation. We don't need to mention it in our docs. Besides, the name of the two clusters can be anything you want, that's why we have the env variables. The instructions are just using `external-cluster` and `remote-cluster` as the example. If that's not obvious I suggest changing lines 65-69:
```
{{< text syntax=bash snip_id=none >}}
$ export CTX_EXTERNAL_CLUSTER=<your external cluster context>
$ export CTX_REMOTE_CLUSTER=<your remote cluster context>
$ export REMOTE_CLUSTER_NAME=<your remote cluster name>
{{< /text >}}
```